banyan/service_infra_helpers.go

```diff
@@ -132,12 +136,11 @@ func flattenExemptions(paths service.ExemptedPaths) (flattened []interface{}, er
 		return
 	}
 
-	if len(paths.Patterns) > 1 {
+	if len(paths.Patterns) != 1 {
```

Comment:
Zero is also not allowed? The error message only talks about "more than one pattern not supported"

Reply:
If you look at previous statement, if value is 0 it does not reach here , I  changed to !=1 for readability purposes